src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        AreaCalculator[] figure = new AreaCalculator[6];
```

Comment:
AreaCalculator should be replaced with Figure as the array should contain Figure objects.